### DISCUSSION TITLE
setting xiSEARCH (DSSO)

### DISCUSSION THREAD

**Discussion by 13651809703** [Rappsilber-Laboratory/XiSearch#65] [Q&A] [answered]
Dear xiSEARCH group,

I am a fresh user of xiSEARCH. Thanks so much for your powerful tool for my CLMS work. Would you kindly help me set xiSEARCH parameters?

1) No amidated and hydrolyzed DSSO were found in variable modifications settings. (version 1.1.6.7). I saw these parameters for BS3.

2) the difference of DSSO(large scale) and DSSO(small scale). For whole proteome-wide, is it right to use DSSO (large scale) to reduce search time?

Thanks a lot. I am looking forward to your reply.

**Comment by lutzfischer** [comment, marked as answer]
DSSO large scale and small scale come with modifications defined as part of the crosslinker.


They are defined in the BasicConfigEntries.conf file 
DSSO(Large Scale)=crosslinker:SymetricSingleAminoAcidRestrictedCrossLinker:Name:DSSO;MASS:158.0037648;LINKEDAMINOACIDS:K(0),S(0.2),T(0.2),Y(0.2),nterm(0);LINEARMODIFICATIONS:NH2,17.026549105,OH,18.0105647,loop,0;STUBS:A,54.0105647,S,103.9932001,T,85.9826354


DSSO(Small Scale)=crosslinker:SymetricSingleAminoAcidRestrictedCrossLinker:Name:DSSO;MASS:158.0037648;LINKEDAMINOACIDS:K(0),S(0.2),T(0.2),Y(0.2),nterm(0);MODIFICATIONS:NH2,17.026549105,OH,18.0105647,loop,0;STUBS:A,54.0105647,S,103.9932001,T,85.9826354


The part that defines the modifications is LINEARMODIFICATIONS:NH2,17.026549105,OH,18.0105647,loop,0 and MODIFICATIONS:NH2,17.026549105,OH,18.0105647,loop,0 respectively. 
Difference being that with peptides containing the as LINEARMODIFICATIONS defined modifications will only be considered as part of linear peptides, while when they are defined as MODIFICATIONS these peptides can also be part of a crosslink.

In both cases the modification mass is the added mass to the crosslinker. E.g. the final dssonh2 modifictaion will have a deltamass of 158.0037648Da + 17.026549105Da = 175.030313905Da.


You can also define the modifications independent of the crosslinker, in the modification section in the BasicConfigEntries.conf file.
DSSO Amidated=modification:[MODE]::SYMBOLEXT:dssonh2;MODIFIED:K,S,T,Y;DELTAMASS:175.030313905
then you can define them as variable or linear in the interface. but then you should change the definition of the crosslinker to not include the modifications.

if you define them independently you could even define that e.g. only the DSSO Amidated(K) is variable while the other are only considered for linear peptide.
DSSO Amidated(K)=modification:[MODE]::SYMBOLEXT:dssonh2;MODIFIED:K;DELTAMASS:175.030313905
DSSO Amidated(S)=modification:[MODE]::SYMBOLEXT:dssonh2;MODIFIED:S;DELTAMASS:175.030313905
DSSO Amidated(T)=modification:[MODE]::SYMBOLEXT:dssonh2;MODIFIED:T;DELTAMASS:175.030313905
DSSO Amidated(Y)=modification:[MODE]::SYMBOLEXT:dssonh2;MODIFIED:Y;DELTAMASS:175.030313905

the same applies for the other modifications.
